ca/proto/generate.go

```diff
@@ -0,0 +1,3 @@
+package proto
+
+//go:generate sh -c "cd ../.. && protoc --go_out=plugins=grpc,Mcore/proto/core.proto=github.com/letsencrypt/boulder/core/proto:. ca/proto/ca.proto"
```

Comment:
I think the `core/proto/core.proto=github.com/letsencrypt/boulder/core/proto` part is not needed because we do not actually import `core.proto` here. I'm guessing this is copied from `va/proto/va.proto`, which does import `core/proto/core.proto`, to get the definitions of Challenge, ValidationRecord, and ProblemDetails.